app/models/review_request.rb

```diff
@@ -39,4 +39,5 @@ class ReviewRequest < ApplicationRecord
                      dependent: :destroy
 
   validates :state, inclusion: { in: states.keys }
+  validates :pull_request_id, uniqueness: { scope: %i[owner_id reviewer_id] }
```

Comment:
👍

Reply:
Does this mean we won't save review requests when the owner of the PR does requested changes and requests a new review?

Reply:
Current_ly we only need the first one right? Or should we save everyone?